### PR TITLE
Bugfix #198.

### PR DESCRIPTION
**Pull request by dpisarenko**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -186,7 +186,7 @@
             <plugin>
                 <groupId>com.jcabi</groupId>
                 <artifactId>jcabi-ssl-maven-plugin</artifactId>
-                <version>0.7.22</version>
+                <version>1.0-SNAPSHOT</version>
```
Comment (krzyk) [inline pom.xml]: we can't use snapshots in the builds

**Comment by krzyk** [comment]
@dpisarenko We use `@NotNull` so we don't have to make ifs for nulls. I think that in this project JSR-303 annotations are not waved, take a look at https://github.com/jcabi/jcabi-maven-plugin/issues/13

**Comment by dpisarenko** [comment]
@krzyk The `@NotNull` annotations don't always work. If you look at the log file fragment of the issue ( https://github.com/yegor256/s3auth/issues/198 ), you see messages like `#withBody(NULL): thrown java.lang.NullPointerException out of com.s3auth.hosts.Resource$PlainText#<init>[125]`. If I understand it correctly, the whole point of this bugfix is to make sure that no NPEs are thrown, even if `@NotNull` doesn't work for some reason.

**Comment by krzyk** [comment]
@dpisarenko I think that first we have to find out why `@NotNull` is not working, I think that it is an issue in configuration.

**Comment by dpisarenko** [comment]
@krzyk How can I get the configuration of the machine, on which that error occured?

File: pom.xml (hunk 1 above)
Comment (dpisarenko) [inline pom.xml]: I fixed this.

**Comment by krzyk** [comment]
@dpisarenko you should ask @yegor256 
You could try to run the application locally, AFAIR it was something like:
`mvn package rexsl:run  -Drexsl.port=9099`
and go to localhost:9099 (or to the specific page that is causing the problems)

**Comment by krzyk** [comment]
@dpisarenko Please close this PR, thanks.

**Comment by davvd** [comment]
@krzyk **19 mins** was added to your account, many thanks for your contribution!; +19 to your rating, your total score is [+2688](http://www.netbout.com/b/35035?open=rating)

**Pull request closed without merging**